src/commands/withdraw.ts

```diff
@@ -1,9 +1,66 @@
+import help from './help';
+import * as utils from '../lib/utils';
+import { getAddressByLabel } from '../integrations/blockIo';
 import { Context } from '../types/bottender';
 
-const withdraw = async (context: Context) => {
-  // TODO get user address
-  // TODO withdraw to address
-  await context.sendText('Ehehehe no way!!');
+export const withdraw = async (context: Context, blockIo: any) => {
+  const fromLabel = context.session.user.id;
+  const fromAddress = await getAddressByLabel(fromLabel);
+  const availableBalance = Number(fromAddress.data.available_balance);
+
+  const [, toAddress, amount] = context.event.text.split(' ');
+  if (!toAddress || !amount || isNaN(+amount)) {
+    help(context);
+    return;
+  }
+
+  if (availableBalance < +amount) {
```

Comment:
I'm not sure, but maybe we need to consider the fees here, or at least see what happens when somebody tries to take out the whole balance...

Reply:
About that, I see 2 possibilities:
1) fees are included in the withdrawal amount and in this case - no problem here
2) fees are excluded, and in this case there should be no minimum
Solution comes probably from block IO, do you have a clue already? We'll try to find out

Reply:
ok, fees are in addition of the specified amounts: https://block.io/docs/basic, which means we have to take them in account

Reply:
kept 2 doge safety when withdrawing, and also got rid of the minimum withdraw amount as fees come extra